modules/reference/pages/cluster-properties.adoc

```diff
@@ -1638,6 +1638,20 @@ Target replication factor for internal topics.
 
 ---
 
+=== minimum_topic_replications
+
+Minimum permitted replication factor value for newly created topics. Redpanda issues an error for any attempt to create a topic with a replication factor less than the specified value. Any existing topics found to have a replication factor less than the specified value will still continue to work as normal. Redpanda will issue a warning level log message at start up indicating the topic's replication factor is less than the specified minimum.
```

Comment:
```suggestion
Minimum permitted replication factor value for newly-created topics. Redpanda returns an error for any attempt to create a topic with a replication factor less than the specified value. Any existing topics found to have a replication factor less than the specified value will continue to work as normal. Redpanda will issue a warning level log message at start up indicating the topic's replication factor is less than the specified minimum.
```

Reply:
When you say "specified value", is that the default of 3? In other words, is the issue that users must stick with the default of 3? Or, can another value be set by a cluster admin and that is the value that is "specified"? I gues I'm wondering if "specified" and "default" are one and the same?